config/traits.yml

```diff
@@ -2586,34 +2586,53 @@ traits:
       allowed_values_max: 4
     pollination_syndrome:
       description: Pollination syndrome
-      notes: Specific animal taxa or abiotic forces that are the plant's primary pollinators (pollen vectors), generally classified based on suites of floral traits. // Specific [NCIT:C38024] animal taxa [NCIT:C25341] or abiotic forces that are the plant's primary pollinators (pollen vectors), generally classified based on suites of floral traits (floral phenotype [NCIT:C16977]).
+      notes: For a given taxon, the biotic vectors (including explicit taxa) or abiotic mechanisms that are most likely to pollinate its flowers, based on the taxon's flower morphology.
       type: categorical
       label: Pollination syndrome
       allowed_values_levels:
-        abiotic: Pollination occurs through wind, water, or collision of free-floating particles
-        animal: Generic catch-all for any form of animal pollination; change to biotic
-        bat: Bat-pollinated
-        bee: Bee-pollinated
-        beetle: Beetle-pollinated
-        bird: Bird-pollinated
-        butterfly: Butterfly-pollinated
-        fly: Fly-pollinated
-        insect: Insect-pollinated, type of insect not specified
-        moth: Moth-pollinated
-        none: Spore-producing plants
-        vertebrate: Vertebrate-pollinated, broadly defined
-        wind: wind-pollinated
-        pollination_rare: plants mostly reproduce through vegetative spread and flowering is rare
+        abiotic: Floral syndrome indicates pollination occurs through wind, water, or collision of free-floating particles.
+        biotic: Floral syndrome suggests flowers are pollinated by a biotic agent.
```

Comment:
These wordings are nice, but they suggest pollination will only ever be inferred from floral syndrome. What if someone observes pollination directly?

Reply:
are, I see trait below has space for field measurements. Would this be a different trait, or rather different value type? When I see the names pollination_syndrome and pollination_system, it doesn't immediately suggest one is observed, the other is inferred.

Reply:
These are the definitions used within the field of pollination biology. See jeffollerton.co.uk/2019/09/05/what-exactly-is-a-pollination-system/

Also, Ruby confirmed these differences between traits are standard in the field and used the same distinction in her dataset.